api/services/AuthorizationPolicy/index.js

```diff
@@ -118,7 +118,10 @@ policy.include('calendarEventInvite', (p) => {
     ({ viewer, entity: { event, status }, action }) =>
       isOrganiser(event, viewer) ||
       isAdmin(viewer) ||
-      (isMember(viewer) && action === ':create'),
+      (isUser(viewer) &&
+        (status === 'Requested' || status === 'Accepted') &&
```

Comment:
Accepted should only be allowed to non admin users when auto_accept_requests is true. 

Correct me if I am wrong. Otherwise add that check here.